currency-rates/internal/handlers/subscribe_test.go

```diff
@@ -10,9 +10,11 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/RinaDish/currency-rates/internal/handlers"
 	"github.com/stretchr/testify/require"
 	"go.uber.org/zap"
+	
+	"github.com/RinaDish/currency-rates/internal/handlers"
+	"github.com/RinaDish/currency-rates/tools"
 )
 
 type successDb struct{}
```

Comment:
The same as client. This is why you need interfaces on level of client usage which you can successfully mock